.github/workflows/codeql-analysis.yml

```diff
@@ -32,14 +32,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'java', 'javascript' ]
+        language: ["java", "javascript"]
         # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
         # Learn more about CodeQL language support at https://git.io/codeql-language-support
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: "17"
+          distribution: "temurin"
```

Comment:
Ik denk nog wel handig om hier de Maven cache aan toe te voegen. Onze Java build is nl een Maven build en dat is wat die CodeQL action ook gebruikt. Dus hier aan toevoegen:

          `cache: maven`
          
    Net als in onze andere GitHub workflow.